src/e2e-test/features/sfmcsource/DesignTimeWithMacros.feature

```diff
@@ -77,3 +77,15 @@ Feature: Salesforce Marketing Cloud Source - Design time Scenarios (macros)
     And Click on the Macro button of Property: "objectName" and set the value to: "objectName"
     And Click on the Macro button of Property: "filter" and set the value to: "filter"
     Then Validate "Salesforce Marketing" plugin properties
+
+  @BATCH-TS-SFMC-DSGN-MACRO-05
+  Scenario: Verify user should be able to validate the plugin when configured for Connection Manager property with macros
+    When Open Datafusion Project to configure pipeline
+    And Select plugin: "Salesforce Marketing" from the plugins list as: "Source"
+    And Navigate to the properties page of plugin: "Salesforce Marketing"
+    And Enter input plugin property: "referenceName" with value: "Referencename"
+    And Click plugin property: "switch-useConnection"
+    And Click on the Macro button of Property: "Connection" and set the value to: "Connection"
+    And Click on the Macro button of Property: "dataExtensionKeyList" and set the value to: "dataExtensionKeyList"
```

Comment:
Check the data-cy for this as well in the source

Reply:
done